tests/app/commands/test_user.py

```diff
@@ -108,7 +108,7 @@ def test_login_command_with_newer_version_available_message(
     assert login_using_api_key_mock.called_once_with(api_key)
     assert access_token_exists.called_once()
     if Version(current_version) < Version(new_version):
-        assert mhandler.SrvOutPutHandler.newer_version_available(new_version) in result.output
+        assert mhandler.SrvOutPutHandler.newer_version_available(new_version, '') in result.output
```

Comment:
Would it be useful if we added a test case for happy path (where `newer_version_available` function returns correct message with clickable download uri)? Not sure if there is another test case that already does this.

Reply:
yeah added link there to validate clickable output as well.